config/axios.js

```diff
@@ -0,0 +1,19 @@
+const axios = require('axios');
+
+const axiosConfig = module.exports;
+
+axiosConfig.initAxios = () => {
+  const getResponse = response => {
+    const { data: responseData } = response;
+    return responseData;
+  };
+
+  const rejectResponse = error => {
```

Comment:
you could log the errors. Check this file: `app/logger`

Reply:
solved